contrib/images/Makefile

```
@@ -1,17 +1,25 @@
 RELAYER_TAG := $(shell grep '^ENV RELAYER_TAG' cosmos-relayer/Dockerfile | cut -f3 -d\ )
+BABYLON_FULL_PATH := $(shell git rev-parse --show-toplevel)
 
 all: babylond cosmos-relayer
 
 babylond: babylond-rmi
-	docker build --tag babylonchain/babylond -f babylond/Dockerfile \
-    		$(shell git rev-parse --show-toplevel)
+	docker build --tag babylonchain/babylond -f babylond/Dockerfile ${BABYLON_FULL_PATH}
+
+babylond-before-upgrade:
+	docker rmi babylonchain/babylond-before-upgrade 2>/dev/null; true && \
+	docker build --tag babylonchain/babylond-before-upgrade -f babylond/Dockerfile \
+    --build-arg VERSION="a33a3344bb44bde2f3374d3cbf919abb942c341a" ${BABYLON_FULL_PATH}
```

Just for clarity:

```suggestion
    --build-arg VERSION="${OLD_VERSION_TAG}" ${BABYLON_FULL_PATH}
```
or so.

---

good idea https://github.com/babylonchain/babylon/pull/727/commits/df5cac90cfb7cf2e28ef2681ddcd3a25cc9a2a19